28.0/Dockerfile

```diff
@@ -0,0 +1,316 @@
+    6E01EEC9656903B0542B8F1003DB6322267C373B \
+    2C7185B3E7454C06D76635694F50789ED785C6D4 \
+    A1A8B25E6185BB18DBAFA60D5F227E08FA339C20 \
+    973730D55793F2DEF00B60DA63B65BEF7899D597 \
+    C25F3295638298E3AFEBE70B48F5EB2E7E59AF8C \
+    B14D3C5CA570F03C911E04A6F0A0ED07755FDBCF \
+    E6DF8502529F86B491C65E1E988BB7626335E3FB \
+    FDE04B7075113BFB085020B57BBD8D4D95DB9F03 \
+    D1DBF2C4B96F2DEBF4C16654410108112E7EA81F \
+    CBE89ED88EE8525FD8D79F1EDB56ADFD8B5EF498 \
+    287AE4CA1187C68C08B49CB2D11BD4F33F1DB499 \
+    4FD6BAE4CAB7261734558654F3BD809A19B82259 \
+    616516B8EB6ED02882FC4A7A8ADCB558C4F33D65 \
+    5C9C707EB7D6FCB75BA1225F6A671EE7D00D38C3 \
+    C388F6961FB972A95678E327F62711DBDCA8AE56 \
+    49DAC1BDFBEB6B26B794904EC08D36845BA9A59F \
+    71A3B16735405025D447E8F274810B012346C9A6 \
+    9DEAE0DC7063249FB05474681E4AED62986CD25D \
+    53D974DA0BAFFF22B3A5FB5C69B4C4CDC628F8F9 \
+    D01B5D68015444D271DAD33FF69705ED890DE427 \
+    1A3E761F19D2CC7785C5502EA291A2C45D0C504A \
+    DB79C612764A580E4BB71BB82B50B0C9E93DB62F \
+    E86AE73439625BBEE306AAE6B66D427F873CB1A3 \
+    45741E495E214C428E8FFF58AC742749DC4965AB \
+    E61773CD6E01040E2F1BD78CE7E2984B6289C93A \
+    CBDB447184AC2CA7A121A27907052EE640FEAC22 \
+    2F78ACF677029767C8736F13747A7AE2FB0FD25B \
+    E58DC1C0A44E0D8E6397D442A54BAA02E69D78D6 \
+    133EAC179436F14A5CF1B794860FEB804E669320 \
+    3F1888C6DCA92A6499C4911FDBA1A67379A1A931 \
+    5ADAB999CE5240DD68F9A079A5210C1621239EFB \
+    AF9167ECC5FB492B9841E276BD991B3EC4EB3A28 \
+    9199E2D71F37DC34BD2337C988AF5B9A92EC92CD \
+    AC6626172E00A82CFFAE8972A636E97631F767E0 \
+    02E82CC36166CAA5F6E623EFB224E825176C0F32 \
+    3EB0DEE6004A13BE5A0CC758BF2978B068054311 \
+    9ED99C7A355AE46098103E74476E74C8529A9006 \
+    0164F03C54C2176A14D1997DFF5C9E3F8ACAF77A \
+    670BC460DC8BF5EEF1C3BC74B14CC9F833238F85 \
+    6A8F9C266528E25AEB1D7731C2371D91CB716EA7 \
+    9343A22960A50972CC1EFD7DB3B5CB8DB648B27F \
+    28E72909F1717FE9607754F8A7BEB2621678D37D \
+    032703671E4ACE58DB7FA2D3BBE369B6789ADBDB \
+    67AA5B46E7AF78053167FE343B8F814A784218F8 \
+    D6D11C357A614D40803ABDC4105F2EA76D0102CA \
+    6EEEF78796B8B5409719A2A36D498CBADF3906B8 \
+    A0083660F235A27000CD3C81CE6EC49945C17EA6 \
+    79D00BAC68B56D422F945A8F8E3A8F3247DBCBBF \
+    B9069CB5D5FB01A66D69E01BC990468AFE0F55CD \
+    8EA063F7D5E6E17ADE409BDC605192A3FE984A60
+
+RUN gpg --keyserver keyserver.ubuntu.com --recv-keys $KEYS
+
+#
+## Option #1: [default] Fetch bitcoind source from release tarballs
+#
+FROM preparer-base AS preparer-release
+
+ARG VERSION
+
+# Download sigs
+ADD https://bitcoincore.org/bin/bitcoin-core-$VERSION/SHA256SUMS.asc ./
+# Download checksums
+ADD https://bitcoincore.org/bin/bitcoin-core-$VERSION/SHA256SUMS ./
+
+# Download source code from same website as github is probably deterministicly built
+ADD https://bitcoincore.org/bin/bitcoin-core-$VERSION/bitcoin-$VERSION.tar.gz ./bitcoin-$VERSION.tar.gz
```

Comment:
Please revert this. Using different sites was intentional to make it harder to perform an attack.

Reply:
Yeah, sorry, I thought I saw a commit in the commit history talking about this.